src/constants.py

```diff
@@ -0,0 +1,4 @@
+# Strikes constants
+MINOR_INFRACTION_POINTS = 1
+MODERATE_INFRACTION_POINTS = 3
+SERIOUS_INFRACTION_POINTS = 7
```

Comment:
hrm... i think since we already have an `enums.py` i think we should rename it to the new `constants.py` and move these three consts into that. can you do that in another PR/make a new ticket for that if you dont feel like doing it now

Reply:
ticket created, MOD-94